src/main/resources/static/templates/JWTVulnerability/keys/public_crt.pem

```diff
@@ -0,0 +1,27 @@
+Bag Attributes
```

Comment:
the cert too

Reply:
same here.

Reply:
hopefully the key and cert are not in active use anywhere :smile:

Reply:
haha yes yes they are not used anywhere else :D